### ISSUE TITLE
How to direct command output to a file

### ISSUE THREAD

**Issue by Pastim** [bassmanitram/actions-for-nautilus#67]
Please forgive me for being dumb, but I can't find a way to make a command work directing the output to a file.  I'm sure it is simple and obvious, but commands I've tried don't work.

So I just want, in one case, to run clamscan (virus checker) and direct output to a file.  I use the normal command line command:

clamscan %F > blahblah.txt

And nothing happens.  Other commands not redirecting stdout seem to work OK.

**Comment by bassmanitram** [comment]
Hey there - what you are doing SHOULD work if you set 'Use shell' to 'true'.

Thanks for using A4N

Martin

**Comment by Pastim** [comment]
Thanks a lot.  That worked.  

I have to admit that I found quite of the amazingly detailed help (most unusual these days!) almost too daunting.  I suspect in this case I have too many other boxes ticked as well (e.g. current working directory, which I seem to have misunderstood), which many have stopped it working when I tried the shell.

Anyway, many thanks for the tool.  I really missed filmanager-actions when I recently ugraded to ubuntu 24.04, so am very glad to have the same (and better) functionality back again.
